runners/sbt/src/main/scala/stryker4s/sbt/SbtMutantRunner.scala

```diff
@@ -31,10 +33,16 @@ class SbtMutantRunner(state: State, processRunner: ProcessRunner, sourceCollecto
     }
   }
 
+  private lazy val emptyLogManager = LogManager.defaultManager(ConsoleOut.printStreamOut(new PrintStream((_: Int) => {})))
```

Comment:
What do you mean with empty log manager?

Reply:
Basically it swallows logging. It doesn't do anything with the logging it receives. Does it need a different name?